monitoring_scripts/http_monitor.py

```diff
@@ -0,0 +1,113 @@
+"""HTTP monitor script main module"""
+
+import argparse
+import logging
+import re
+import requests
+
+from . import nagios_common as nagios
+
+parser = argparse.ArgumentParser(
+    prog='http-monitor',
+    description='Nagios-compatible plugin to check a resource availability over HTTP/HTTPS. By '
```

Comment:
I'd recommend putting that description as the `http_monitor.py` module's docstring and use `__doc__` as value for that keyword argument.